index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="zxx">
+    <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Trial-project1</title>
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Virgil</h1>
+    </main>
+    <figure id = "img-div">
+      <img  id = "image" src = "https://r.lvmh-static.com/uploads/2021/11/vaok_site-news-cover-1000x550.jpg" alt="image of Virgil">
+      <p id="img-caption">Vigil posing for Time Magazine in a past interview</p>
+    </figure>
+    <p id="tribute-info">
+      Virgil was an American fashion designer and entrepreneur. He was the artistic director of Louis Vuitton's menswear collection beginning in 2018, and was given increased creative responsibilities across the brand in early 2021.
+    </p>
```

Comment:
- [ ] Kindly note to always include an explanation, a "why", for each requested change. For example:

- This is a well-explained request:
  - It would be better to not have linters errors before requesting a review. A linter will help avoid errors, and things that could not be errors (but look like them) or could be potentially dangerous.
- This request lacks a "why":
  - Please, ensure that you do not have linters errors before requesting for a review.
- [ ] Remember that the comment should be a suggestion instead of making it look like an order.

Reply:
It would be better to ensure that there are no linters errors before requesting a review as linter will help avoid errors, and anything that could be potentially dangerous /harmful.